images/kube-deploy/Dockerfile

```diff
@@ -0,0 +1,31 @@
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Includes go, gcloud, and etcd
+FROM gcr.io/k8s-testimages/gcloud-in-go:v20171113-192bec25
```

Comment:
that gcloud-in-go tag pins gcloud to 163.0.0, is that what you want?

Reply:
I don't really care. I only need gcloud to work with the test-infra stuff, which I think for these jobs is just log uploading. I chose this because our build already runs fine in it, just the integration tests need etcd.